### PR TITLE
Message Preview + Conversation List stories + components

### PR DESCRIPTION
**Pull request by daria-github**
Relates to #98 

**Message preview:**
Screenshot 2023-03-02 at 9 37 51 AM

**Message preview loading**
Screenshot 2023-03-02 at 9 37 58 AM

**Conversation list**
Screenshot 2023-03-02 at 9 38 12 AM

**Conversation list loading**
Screenshot 2023-03-02 at 9 38 18 AM

**Conversation list loaded but no messages**
Screenshot 2023-03-02 at 9 38 25 AM

### CODE REVIEW

**Comment by m-005** [comment]
Relates to #98

File: component-library/components/MessagePreviewCard/MessagePreviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Avatar } from "../Avatar/Avatar";
+import moment from "moment";
```
Comment (rygine) [inline component-library/components/MessagePreviewCard/MessagePreviewCard.tsx:3]: `moment` is a very heavy library, especially if we're using it like this. we should consider `date-fns` or `luxon` instead.

also, `moment` is now in [maintenance mode](https://momentjs.com/docs/#/-project-status/).

File: component-library/components/MessagePreviewCard/MessagePreviewCard.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   */
+  onClick?: () => void;
+  // To-do: Add error views once we have the designs
+}
+
+export const MessagePreviewCard = ({
+  text = "New message",
+  avatar = <Avatar />,
+  displayAddress = "New recipient",
+  datetime = moment(new Date()).fromNow(),
```
Comment (rygine) [inline component-library/components/MessagePreviewCard/MessagePreviewCard.tsx:41]: check out [formatDistanceToNow](https://date-fns.org/v2.29.3/docs/formatDistanceToNow) for a `date-fns` equivalent.

File: component-library/components/MessagePreviewCard/MessagePreviewCard.tsx (hunk 1 above)
Comment (daria-github) [reply component-library/components/MessagePreviewCard/MessagePreviewCard.tsx:3]: it is heavy, but i felt the drawbacks weren't too significant for our use case — i've not used date-fns but will take a look and try to make the update. thanks for the suggestion!

File: component-library/components/MessagePreviewCard/MessagePreviewCard.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {isLoading ? (
+          <ShortCopySkeletonLoader />
+        ) : (
+          <span className="text-md text-gray-500">{text}</span>
+        )}
+      </div>
+      {isLoading ? (
+        <IconLoader />
+      ) : (
+        <div className="text-xs text-gray-400 w-1/4 text-right">{datetime}</div>
```
Comment (rygine) [inline component-library/components/MessagePreviewCard/MessagePreviewCard.tsx:65]: just wanted to note here that the `formatDistanceToNow` function doesn't include `ago` in the output like `moment` did. if we still want to keep that, i think just adding it afterwards manually would be fine.

Comment (daria-github) [reply component-library/components/MessagePreviewCard/MessagePreviewCard.tsx:65]: thanks for flagging that! let's leave it with the default text for the time being until we get product direction

**Review by rygine — APPROVED** [review]

**Review by bhavya2611 — APPROVED** [review]
LG

**Comment by daria-github** [comment]
Closing, this change is now lumped in with some WIP changes

**Pull request closed without merging**